inference-engine/cmake/dependencies.cmake

```diff
@@ -133,6 +133,18 @@ if (THREADING STREQUAL "OMP")
     
 endif ()
 
+## pre-production package for hybrid support (windows/macos/linux only)
+if (NOT ANDROID AND NOT(LINUX AND AARCH64))
+    if(DEFINED ENV{THIRDPARTY_SERVER_PATH})
+        set(IE_PATH_TO_DEPS "$ENV{THIRDPARTY_SERVER_PATH}")
+    elseif(DEFINED THIRDPARTY_SERVER_PATH)
+        set(IE_PATH_TO_DEPS "${THIRDPARTY_SERVER_PATH}")
+    else()
+        message(WARNING "THIRDPARTY_SERVER_PATH is not set (env or cmake cmd-line define). Pre-production TBB (with Hybrid support) cannot be loaded!")
+    endif()
+    message(STATUS "THIRDPARTY_SERVER_PATH=${IE_PATH_TO_DEPS}")
+endif()
```

Comment:
setting this globally, overrides GNA default downloading paths

Reply:
looks like the proper way is to upload to download01.org

CC @asuhov

Reply:
@asuhov, @ilya-lavrenov isn't a proper way is just to add the GNA package to the testing server? Can someone help me with it?

Reply:
The package was updated.